internal/cloudapi/v2/handler.go

```diff
@@ -365,6 +363,22 @@ func (h *apiHandlers) PostCompose(ctx echo.Context) error {
 				t.ImageName = key
 				t.OsbuildArtifact.ExportFilename = imageType.Filename()
 
+				irTarget = t
+			case ImageTypesEdgeContainer:
```

Comment:
@Gundersanne Are we fine with breaking the backward compatibility here?

Reply:
I was also concerned about this. But, we had a discussion about it a few weeks ago and I don't remember who said that edge-containers are not really used currently, so breaking it should be OK. But, yes, this change makes old Workers not be able to build edge-container configured by a newer composer service

Reply:
It's true that the image type isn't used atm. This does break the image type, as configuration won't be in place when this PR lands, and I do not want to block releasing on an unused image type.

Reply:
Approved PR, I agree, this is fine as-is.